code/modules/organs/organ_external.dm

```diff
@@ -161,8 +156,16 @@
 	brute *= brute_mod
 	burn *= burn_mod
 
+	// Threshold needed to have a chance of hurting internal bits with something sharp
+	var/const/sharp_thresh_int_dmg = 5
+	// Threshold needed to have a chance of hurting internal bits
+	var/const/thresh_int_dmg = 10
+	// Probability of taking internal damage from sufficient force, while otherwise healthy
+	var/const/dmg_prob = 5
+	// Probability of taking internal damage when hit while lacking encasing bones
+	var/const/no_bone_dmg_prob = 30
```

Comment:
I mean, both effectively eliminate the problem of having magic numbers, it's just that #defines seem to be used moreso than constant variables, probably since they're easier to find/are usually just placed right at the beginning of the file.